cpp/src/strings/utilities.cu

```diff
@@ -159,12 +181,13 @@ special_case_mapping* d_special_case_mappings           = nullptr;
 const character_flags_table_type* get_character_flags_table()
 {
   std::lock_guard<std::mutex> guard(g_flags_table_mutex);
-  if (!d_character_codepoint_flags) {
```

Comment:
This does not look right.
With the if-statement removed, the symbol copy will happen every time this is called.
The intention was to only do the copy once.

Reply:
d_character_codepoint_flags is now a cache rather than a raw pointer. So this calls the cache function now every time, but usually it's just a successful `map` lookup.

Reply:
Ok, I missed the lambda being called inside the if inside find_or_initialize.